update(getting started): fill out the todo, fix first page (#1623)
bonitasoft/bonita-doc: modules/ROOT/pages/tutorial-overview.adoc

== Before starting

> . Make sure to check out the page that tells you everything about xref:what-is-bonita.adoc[What Bonita is]. +

Reviewer: Which page(s) ?? A link please :)

Author: Made the link clearer

> . Make sure to check out the page that tells you everything about xref:what-is-bonita.adoc[What Bonita is]. +

Author: ```suggestion
. Make sure to xref:what-is-bonita.adoc[check this chapter], that tells you everything about what Bonita is. +
```
